Implement the 3d vector subtraction test

MarcelWolkober opened the pull request:
The 3D Vector Subtraction test was implemented by adding a basic test and asserting that the three values are correct.

MarcelWolkober committed:
Implement the 3d vector subtraction test

heidrifx commented on exercise1.py:
```diff
@@ -46,4 +46,6 @@ def test_3d_vector_addition() -> None:
-    assert False
+    v = Vector([1.1, 2.2, 3.3]) - Vector([1.0, 2.0, 3.0])
+    
+    assert isclose(v[0], 0.1)
```
Please also check the other coords!

MarcelWolkober committed:
Add missing assert.

Added the missing asserts which were not correctly added to the commit ( VSCode fault)

heidrifx reviewed (approved):
lgtm
